lib/ffmpeg/encoderbase.py

```diff
@@ -239,14 +240,14 @@ def check_bitrate(self):
       size_encoded = encsize,
       bitrate_actual = "{:-.2f}".format(bitrate_actual))
 
-    if "cbr" == self.rcmode:
+    if "cbr" == self.rcmode and "bitrate_gap" not in self.skip_checks:
```

Comment:
Use `vars(self).get("skip_checks", [])`, instead, here and in the other places... because this is an optional parameter.

Reply:
Thank you for your review, I have made corresponding modifications to it